### ISSUE TITLE
Text task

### ISSUE THREAD

**Issue by srallen** [zooniverse/front-end-monorepo#1141]
**Package** lib-classifier

**Feature or Issue Description**
The text task needs to be added w/ the buttons for inserting the metadata values like deletion

**To Dos**

- [x] Simple text task using a textarea element
- [x] Metadata buttons to insert metadata values

**Comment by srallen** [comment]
The metadata values are optional and configurable in the project builder, none or only one or two of them may be toggled.

**Comment by srallen** [comment]
@beckyrother I think we need a design for how this looks with the metadata buttons. Here's what it looks like in PFE currently: 

https://master.pfe-preview.zooniverse.org/projects/srallen086/untitled-project-5-10-2018-11-53-20-am-yep/classify

**Comment by eatyourgreens** [comment]
We should post an example of the task description JSON, from a workflow, here.

**Comment by srallen** [comment]
Good idea. Here's a sample from my test project:

``` json
{
  "annotations": [
    {
      "task": "T0",
      "value": "this is my text [insertion]insertion[/insertion].[deletion]deletion[/deletion][unclear]this is unclear[/unclear]."
    }
  ]
}
```

 I don't think the JSON structure of these annotation types will be changing, but I'll ping @CKrawczyk to let us know if we should consider any changes to help with aggregation.

**Comment by eatyourgreens** [comment]
Thanks, but I was asking for the task description from a workflow's tasks array.

**Comment by srallen** [comment]
Oh shoot, my bad. Here's what that looks like:

``` json
{ "tasks": 
  {
    "T0": {
      "help": "",
      "instruction": "Enter an instruction.",
      "required": false,
      "text_tags": ["unclear", "insertion", "deletion"],
      "type": "text"
    }
  }
}
```

All of these are the defaults that the lab creates expect for the `text_tags` array, which I enabled so we could see them.

**Comment by beckyrother** [comment]
Agreed, I'll make something. Is tomorrow ok?

**Comment by srallen** [comment]
Yes, that's ok. Thank you.

**Comment by beckyrother** [comment]
Text modifiers: https://projects.invisionapp.com/d/main/#/console/12923997/270495290/inspect

**Comment by beckyrother** [comment]
They're using the Text Link style, same as 'Need some help with this task?'

**Comment by beckyrother** [comment]
Right. Just talked to Sam about ASM and ways we can adapt for FER, meeting with @mcbouslog and @simensta on Friday to talk about that portion. I have a few outstanding questions; @srallen would you like to be part of the call Friday?

**Comment by eatyourgreens** [comment]
The original text task resizes as you type. I've ignored that in #1232.
https://github.com/zooniverse/Panoptes-Front-End/blob/5de8c9962f753d4443a4f5d346df125b78de4b3a/app/classifier/tasks/text/index.jsx#L88-L96

I've also ignored the debounced inputs in the original code, which were a workaround for bad performance when updating the classification after each keypress. I think the new text task should not update the classification (or drawing mark for subtasks) until you've finished typing.
